test/unit/spec/Util/BootstrapDialogSpec.js

```diff
@@ -72,5 +78,54 @@ define (
             simpleDialog.setBody($newBody);
             expect(simpleDialog.getBody().html()).toBe($newBody.html());
         });
+
+        it('Should trigger a command on enter key', function() {
+            var wasTriggered = false;
+            var btnFn = function() {
+                wasTriggered = true;
+            };
+            var dialog = new Dialog({
+                title: 'test',
+                body: $('<div>'),
+                buttons: [$('<button>').append('foo').click(btnFn)],
+                enterToTrigger: true
+            });
+            dialog.show();
+            expect(wasTriggered).toBe(false);
+            var e = $.Event('keypress');
+            e.which = 13;
+            e.keyCode = 13;
+            dialog.getElement().trigger(e);
+            expect(wasTriggered).toBe(true);
+            dialog.hide();
+            dialog.destroy();
+        });
+
+        it('Should create a simple alert', function() {
+            var alert = new Dialog({
+                title: 'Alert',
+                body: $('<div>').append('an alert'),
+                alertOnly: true
+            });
+
+            var btns = alert.getButtons();
+            expect(btns.length).toBe(1);
+            expect(btns[0].innerHTML).toBe('Close');
+        });
+
+        it('Should set the title based on type', function() {
+            var d = new Dialog({
+                title: 'Foo',
```

Comment:
I'd recommend making this something more user-friendly. I know that the title should be set based on the type, but just in case there's some odd condition that cause it not to be set...maybe make the default either a blank string or something like "Title should have been set by type"